### ISSUE TITLE
max-tags not respected when using separators

### ISSUE THREAD

**Issue by jmillard-optimo** [JohMun/vue-tags-input#28]
When using the following `vue-tags-input`: 

```
<vue-tags-input
   class="tags-input"
   v-model="tag"
   :tags="tags"
   :allow-edit-tags="true"
   :add-from-paste="false"
   :avoid-adding-duplicates="false"
   :delete-on-backslash="true"
   :max-tags="5"
   :separators="[',']"
   :placeholder="Enter up to 5 tags, separated by commas."
   @tags-changed="function(newTags) { return tags = newTags; }"
   @before-adding-tag="function(obj) { obj.tag.text = obj.tag.text.trim(); if (obj.tag.text.length > 0) { obj.addTag(); }}"/>
```

If I type:

`1`_enter_
`2`_enter_
`3`_enter_
`4`_enter_

Followed by: 

`5,6,7,8,9,10`_enter_

![screen shot 2018-08-10 at 9 03 38 am](https://user-images.githubusercontent.com/40368837/43959311-5ae01b7e-9c7c-11e8-99fc-4a5656650271.png)

I will get 10 tags:

![screen shot 2018-08-10 at 9 03 45 am](https://user-images.githubusercontent.com/40368837/43959321-642c141c-9c7c-11e8-9e7a-2050ceabc3da.png)

From that point on, I can add as many tags as I want.

I can also just enter `1,2,3,4,5,6,7,8,9,10` and get the same effect.

**Comment by JohMun** [comment]
thx for reporting this! It should be fixed in [v1.5.5](https://github.com/JohMun/vue-tags-input/releases/tag/v1.5.5).

**Comment by jmillard-optimo** [comment]
Implemented newer version and tested. Worked great!  Thanks!
